pennylane/ops/op_math/symbolicop.py

```diff
@@ -75,6 +75,18 @@ def base(self) -> Operator:
         """The base operator."""
         return self.hyperparameters["base"]
 
+    # pylint: disable=protected-access
+    def _check_batching(self, params):
+        self.base._check_batching(params)
+
+    @property
+    def batch_size(self):
+        return self.base.batch_size
+
+    @property
+    def ndim_params(self):
+        return self.base.ndim_params
+
```

Comment:
Do we want additional tests covering these functionalities for `SymbolicOp` separately, or is their coverage via tests on its child classes sufficient? :)